ui/raidboss/data/06-ew/trial/rubicante.ts

```diff
@@ -0,0 +1,284 @@
+//  - Flamerake - identify initial direction of line cleave?
+
+export interface Data extends RaidbossData {
+  decOffset?: number;
+  dualfireTarget: boolean;
+}
+
+// First headmarker is Shattering Heat (tankbuster on MT)
+const firstHeadmarker = parseInt('0156', 16);
+const getHeadmarkerId = (data: Data, matches: NetMatches['HeadMarker']) => {
+  if (typeof data.decOffset === 'undefined')
+    data.decOffset = parseInt(matches.id, 16) - firstHeadmarker;
+  return (parseInt(matches.id, 16) - data.decOffset).toString(16).toUpperCase().padStart(4, '0');
+};
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.MountOrdeals,
+  timelineFile: 'rubicante.txt',
+  initData: () => {
+    return {
+      dualfireTarget: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'Rubicante Headmarker Tracker',
+      type: 'HeadMarker',
+      netRegex: {},
+      condition: (data) => data.decOffset === undefined,
+      run: (data, matches) => {
+        getHeadmarkerId(data, matches);
+      },
+    },
+    {
+      id: 'Rubicante Inferno Raidwide',
+      type: 'StartsUsing',
+      netRegex: { id: '7CEA', source: 'Rubicante', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Rubicante Ordeal of Purgation',
+      type: 'StartsUsing',
+      netRegex: { id: ['7CC4', '80E8'], source: 'Rubicante', capture: false },
+      alertText: (_data, _matches, output) => output.avoid!(),
+      outputStrings: {
+        avoid: {
+          en: 'Avoid cone',
+        },
+      },
+    },
+    {
+      id: 'Rubicante Shattering Heat',
+      type: 'StartsUsing',
+      netRegex: { id: '7CEB', source: 'Rubicante' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Rubicante Blazing Rapture',
+      type: 'StartsUsing',
+      netRegex: { id: '7CD2', source: 'Rubicante', capture: false },
+      // This is a ~14 second cast.
+      delaySeconds: 5,
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'Rubicante Inferno Spread',
+      type: 'StartsUsing',
+      netRegex: { id: '7CDA', source: 'Rubicante' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Rubicante Flamerake',
+      type: 'StartsUsing',
+      netRegex: { id: '7CDD', source: 'Rubicante', capture: false },
+      // Spinning compass animation is 6 seconds; damage goes out 2.6 seconds later
+      delaySeconds: 3,
+      infoText: (_data, _matches, output) => output.avoid!(),
+      outputStrings: {
+        avoid: {
+          en: 'Avoid line cleave, then in',
+        },
+      },
+    },
+    {
+      id: 'Rubicante Soulscald',
+      type: 'Ability',
+      netRegex: { id: '7CE6', source: 'Rubicante', capture: false },
+      delaySeconds: 3,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Bait protean, then dodge',
```

Comment:
I'm not sure this necessarily needs a trigger, because there's ground markers for this that show up with plenty of time to dodge.  If you wanted to have a prompt to get out of the way, maybe just a `moveAway` when the initial damage hits, like this similar mechanic? https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/05-shb/trial/hades-ex.ts#L61-L67